common/lib/xmodule/xmodule/course_module.py

```diff
@@ -834,6 +834,16 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    social_sharing_url = String(
+        display_name=_("Social Media Sharing URL"),
+        help=_(
+            "A fully-qualified URL used by the social sharing icons on the dashboard (if enabled) "
```

Comment:
@mattdrayer @marcotuts who enters these URLs/who is the audience for this help text? It would be great to provide an example in the help text. I'll catch up with one or other of you re the workflow, so that I'm more informed before making suggestions here.